### ISSUE TITLE
Random "unexpected end of JSON input" errors

### ISSUE THREAD

**Issue by galderz** [infinispan/infinispan-operator#262]
[Example](https://travis-ci.org/infinispan/infinispan-operator/jobs/637474531?utm_medium=notification&utm_source=github_status)

```
--- FAIL: TestClusterFormationWithTLS (389.59s)
panic: timed out waiting for the condition, last error: unable to decode: unexpected end of JSON input [recovered]
	panic: timed out waiting for the condition, last error: unable to decode: unexpected end of JSON input
goroutine 2506 [running]:
testing.tRunner.func1(0xc000100f00)
	/home/travis/.gimme/versions/go1.13.6.linux.amd64/src/testing/testing.go:874 +0x3a3
panic(0x11c57c0, 0xc000647820)
	/home/travis/.gimme/versions/go1.13.6.linux.amd64/src/runtime/panic.go:679 +0x1b2
github.com/infinispan/infinispan-operator/test/e2e/util.ExpectNoError(...)
	/home/travis/gopath/src/github.com/infinispan/infinispan-operator/test/e2e/util/asserts.go:8
github.com/infinispan/infinispan-operator/test/e2e.waitForPodsOrFail(0xc00095d000, 0x13a0387, 0x5, 0x2)
	/home/travis/gopath/src/github.com/infinispan/infinispan-operator/test/e2e/main_test.go:404 +0x2e4
github.com/infinispan/infinispan-operator/test/e2e.TestClusterFormationWithTLS(0xc000100f00)
	/home/travis/gopath/src/github.com/infinispan/infinispan-operator/test/e2e/main_test.go:123 +0x20a
testing.tRunner(0xc000100f00, 0x142d800)
	/home/travis/.gimme/versions/go1.13.6.linux.amd64/src/testing/testing.go:909 +0xc9
created by testing.(*T).Run
	/home/travis/.gimme/versions/go1.13.6.linux.amd64/src/testing/testing.go:960 +0x350
FAIL	github.com/infinispan/infinispan-operator/test/e2e	607.082s
```

**Comment by dmvolod** [comment]
@galderz looks like fix #280 is resolved this issue

**Comment by rigazilla** [comment]
@dmvolod I agree. Closing, thanks.
